lib/plugins/create/templates/aws-scala-sbt/project/plugins.sbt

```diff
@@ -1,4 +1,3 @@
-resolvers += Resolver.sonatypeRepo("public")
-
 addSbtPlugin("com.eed3si9n" % "sbt-assembly" % "0.14.9")
+addSbtPlugin("com.github.nomadblacky" % "sbt-assembly-log4j2" % "0.1.1")
```

Comment:
A sbt-plugin has been released that merges `Log4j2Plugins.dat`.

+ https://github.com/idio/sbt-assembly-log4j2

However, it doesn't support sbt 1.x.  
I released a fork that supports sbt 1.x.

+ https://github.com/NomadBlacky/sbt-assembly-log4j2